microsoft-edge/devtools-guide-chromium/css/index.md

```diff
@@ -156,27 +156,29 @@ Using separate layouts allows for a one-column layout for mobile devices and mul
 
 Use the following steps to debug or test the media queries that you defined in your CSS.
 
-1. To open DevTools, you can right-click in a webpage and then select **Inspect**.
+1. Open the [CSS Examples](https://microsoftedge.github.io/Demos/devtools-css-get-started/) demo page in a new window or tab.
+
+1. To open DevTools, right-click in the webpage and then select **Inspect**.
 
 1. Click the **Toggle device emulation** (![Device Emulation icon](./index-images/device-emulation-icon-light-theme.png)) button.  Or, when DevTools has focus, press **Ctrl+Shift+M** (Windows, Linux) or **Command+Shift+M** (macOS).
 
-   <!-- todo: update to show new tooltip: -->
+   The Device Toolbar opens in the webpage, and the webpage is now rendered within the Device Emulation pane:
 
-   ![Opening the device toolbar](./index-images/css-elements-media-queries-open-device-toolbar.png)
+   ![Opening the device toolbar](./index-images/media-queries-open-device-toolbar.png)
 
-1. With the device toolbar open, click the **...** menu on the top-right and then select **Show media queries**:
+1. With the device toolbar open, click the **More options** (![The more options icon](./index-images/more-options-icon.png)) button on the top-right, and then select **Show media queries**:
 
-   ![Showing media queries in the Device Toolbar](./index-images/css-elements-media-queries-showing-mq.png)
+   ![Showing media queries in the Device Toolbar](./index-images/media-queries-showing-mq.png)
 
    The colored bars above the webpage represent the different media queries.
        
 1. Hover on the boundaries in the bars to display the values of the different media queries.  Click each media query value to resize the web page to match.
```

Comment:
global in .md: change "hover on" to "hover over" per
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/term-collections/mouse-mouse-interaction-terms